client/components/About/About.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+
+import styles from './About.scss'
+
+class About extends Component {
+  render () {
+    return (
+      <section>
```

Comment:
Let's add a `aboutSection` class with
```
.aboutSection {
  padding: 0 60px;
  display: grid;
}
```
to keep consistent padding with the other pages